bin/test-workflow/utils.sh

```diff
@@ -254,6 +254,15 @@ function split_on_comma_delimiter {
   echo "${array[@]}"
 }
 
+function uninstall_helm_release {
+release_name=$1
```

Comment:
Function body should be indented.
`$1` and `$2` should have quotes around them.